hack/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Is this hack directory typical? First time I'm seeing it. Looks like a good idea